doc/tutorials/workshop/content/exercises/10-rule-parametrization.md

```diff
@@ -0,0 +1,182 @@
+---
+Title: Rule parametrization
+PrevPage: 09-writing-rules
+NextPage: 12-node-rules
+---
+
+Rule parametrization
+====================
+
+In this section we will learn how one can customize their rules with
+parametrization. And how to test these rules with more advanced procedures.
+
+## Variable parametrization
```

Comment:
or say `Using tailored variables in rules` here instead of last comment

Reply:
I rephrased the section a bit.